src/controller/handler/issue.credential.ts

```diff
@@ -138,3 +138,26 @@ export class IssueCredential extends BaseAgentResponseHandler {
         return;
     }
 }
+
+interface Body {
```

Comment:
same.  I do not understand why we are creating interfaces here.

Reply:
Yeah, it might come from an external source, but that external source needs to obey the contract defined by aries-controller. Let's pull this out as a DTO and add some validations to it.